py/strato/common/log/log2text.py

```diff
@@ -288,19 +301,20 @@ def runRemotely(host, ignoreArgs):
                                           hostname=hostname)
 
     sshCommand = 'sshpass -p %(password)s ssh -X -f -o ServerAliveInterval=5 -o ServerAliveCountMax=1 -o ' \
-                 'StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null %(host)s %(command)s | less -r'
+                 'StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=10 ' \
```

Comment:
I know it was my idea, but please verify whether ConnectTimeout its only on the connection or on the whole operation.